content/tracing/languages/java.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Tracing Java Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-java"
+  tag: "Github"
+  text: Source code
+- link: "/tracing/"
+  tag: "Documentation"
+  text: APM (Tracing)
+---
+
+## Compatibility
+
+We officially support Java 1.7 and higher of both Oracle JDK and OpenJDK.  To see what web frameworks, libraries, and datastores we support, see the Integrations section below.
+
+## Installation and Getting Started
+
+To begin tracing applications written in any language, first [install and configure the Datadog Agent](https://docs.datadoghq.com/tracing#installing-the-agent).
+
+Next, download `dd-java-agent.jar` that contains the agent class files:
+
+```shell
+wget -O dd-java-agent.jar 'https://search.maven.org/remote_content?g=com.datadoghq&a=dd-java-agent&v=LATEST'
+```
+
+Finally, add the following JVM argument when starting your application—in your IDE, your Maven or Gradle application script, or your `java -jar` command:
```

Comment:
Why is `application—in` hyphenated? Should it be `application in`?

Reply:
I think that should have been a double dash with a space on both sides. I’m not sure what the correct punctuation should be though.

Reply:
👍